presto-parser/src/test/java/com/facebook/presto/sql/parser/TestSqlParser.java

```diff
@@ -2271,6 +2274,142 @@ public void testDropRole()
         assertStatement("DROP ROLE \"!@#$%^&*'ад\"\"мін\" IN \"カタログ\"", new DropRole(new Identifier("!@#$%^&*'ад\"мін"), Optional.of(new Identifier("カタログ"))));
     }
 
+    @Test
+    public void testGrantRoles()
```

Comment:
some test with CURRENT_USER / CURRENT_ROLE?

Reply:
ignored for now